src/main/java/org/opensearch/security/support/ConfigConstants.java

```diff
@@ -320,6 +320,9 @@ public enum RolesMappingResolution {
     public static final String TENANCY_GLOBAL_TENANT_NAME = "global";
     public static final String TENANCY_GLOBAL_TENANT_DEFAULT_NAME = "";
 
+    public static final String EXTENSIONS_BWC_PLUGIN_MODE = "bwcPluginMode";
```

Comment:
`BWC_PLUGIN_MODE` may not be the best name to capture what this setting does. In an early draft of security for extensions it was called this because it was not fully known yet what it would take for an extension to be backward compatible with plugins. 

wdyt about calling this setting `EXTENSIONS_INCLUDE_BACKEND_ROLES`?

Reply:
I just initiated the run of CI. I doubt the naming including 'EXTENSIONS' gonna pass the lint task, since we do have restrictions of using it.

Reply:
`jwtTokenIncludesBackendRoles` ?

Reply:
I think that should be fine since this mode is targeting for the usage of the extension.